funsite/static/funsite/css/header.css

```diff
@@ -41,6 +41,21 @@
     font-family: 'Raleway', sans-serif;
 }
 
+#top-menu span.fun-logo {
+    background-image: url('../images/logos/fun61.png');
```

Comment:
I noticed, see it more like a proof of concept, I'll take care of this after your merge